features/project/source/browse_files.feature

```diff
@@ -133,6 +143,16 @@ Feature: Project Source Browse Files
     Then I am redirected to the files URL
     And I don't see the ".gitignore"
 
+  @javascript
+  Scenario: I can replace file and commit
+    Given I click on ".gitignore" file in repo
+    And I see the ".gitignore"
+    And I click on "Replace"
+    When I replace it with "LICENSE"
+    And I fill the commit message
+    And I click on "Replace file"
+    Then I am redirected to the ".gitignore"
+
```

Comment:
@tsigo @stanhu I let the test to check the content of current page after click "Upload file" button contains "user.feature", then the upload is successful, do you think is this fine?
The previous feature I think is that variable current_path is not the path to current page after upload, so it is not consistent.
![chinesereminder1](https://cloud.githubusercontent.com/assets/7370334/8355111/94561bc8-1b4d-11e5-96b2-ede69b117f67.png)